packages/edit-site/src/components/layout/style.scss

```
@@ -140,6 +142,14 @@
 	}
 }
 
+.edit-site-layout__canvas:has(.is-oversized) {
```

I am uneasy with breaking component boundaries in general, but at the very least I would add a class selector here for safety. (I know the content itself is iframed but any of our intermediary components could have the same modifier class.)

```suggestion
.edit-site-layout__canvas:has(.edit-site-the-frame__content.is-oversized) {
```